lib/split/algorithms/whiplash.rb

```diff
@@ -1,38 +0,0 @@
-# frozen_string_literal: true
-# A multi-armed bandit implementation inspired by
-# @aaronsw and victorykit/whiplash
-require 'simple-random'
-
-module Split
-  module Algorithms
-    module Whiplash
```

Comment:
Since Whiplash relies on SimpleRandom, and we don't use Whiplash (we use Weighted Sample), we have decided to remove this module entirely. It can be re-instated in the future if we decide we need to use the Whiplash algorithm (with a replacement for SimpleRandom).